components/cloud-foundry/src/model/build-pack.model.js

```diff
@@ -0,0 +1,70 @@
+(function () {
```

Comment:
I'd originally added to support selecting a build pack in a manifest step. Will remove